util/Bigfork/Installer/Install.php

```diff
@@ -133,7 +133,7 @@ public static function postUpdate(Event $event)
     protected static function applyConfiguration(array $config)
     {
         $base = self::getBasepath();
-        include $base.'/framework/thirdparty/spyc/spyc.php';
+        include $base.'/vendor/mustangostang/spyc/spyc.php';
```

Comment:
Turns out that composer’s autoloader _can_ do the magic for us. Can you remove this `include` statement, and then add  `require 'vendor/autoload.php';` to the top of the file? (just after the `use` statements)